### PR TITLE
Upgrade IntelliJ Idea with bundled JDK to 14.1

### PR DESCRIPTION
**Pull request by sebroeder**

### CODE REVIEW

File: Casks/intellij-idea-bundled-jdk.rb [hunk 1]
```diff
@@ -12,5 +12,7 @@
   zap :delete => [
                   '~/Library/Application Support/IntelliJIdea14',
                   '~/Library/Preferences/IntelliJIdea14',
+                  '~/Library/Caches/IntelliJIdea14',
```
Comment (vitorgalvao) [inline Casks/intellij-idea-bundled-jdk.rb]: These could be improved in the same way as https://github.com/caskroom/homebrew-versions/pull/877/files#r27178699.

Comment (sebroeder) [inline Casks/intellij-idea-bundled-jdk.rb]: > These could be improved in the same way as https://github.com/caskroom/homebrew-versions/pull/877/files#r27178699.

Done. Thanks for the hints.

**Comment by sebroeder** [comment]
Please do not merge yet, see discussion in #877 for details.

**Comment by sebroeder** [comment]
This PR can me merged now as far as I am concerned.

**Comment by radeksimko** [comment]
LGTM. :+1:

**Pull request merged**